pandas/tests/extension/test_sparse.py

```diff
@@ -385,7 +382,7 @@ def test_equals(self, data, na_value, as_series, box):
 
 
 class TestCasting(BaseSparseTests, base.BaseCastingTests):
-    def test_astype_object_series(self, all_data):
+    def _test_astype_object_series(self, all_data):
```

Comment:
Don't think these tests should start with `_`?

Reply:
good catch, this test should just be removed and we can use the base class test